src/lib/swapper/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.ts

```diff
@@ -75,19 +73,6 @@ export async function getCowSwapTradeQuote(
   )
 
   if (maybeQuoteResponse.isErr()) {
-    const err = maybeQuoteResponse.unwrapErr()
-    const errData = (err.cause as AxiosError)?.response?.data
-    if (
-      (err.cause as AxiosError)?.isAxiosError &&
-      errData?.errorType === 'SellAmountDoesNotCoverFee'
-    ) {
-      return Err(
-        createTradeAmountTooSmallErr({
-          assetId: sellAsset.assetId,
-          minAmountCryptoBaseUnit: bn(errData?.data.fee_amount ?? '0x0', 16).toFixed(),
-        }),
-      )
-    }
```

Comment:
Was this removal intentional?

It causes us to no longer show the CoW swapper when the sell amount is too low, e.g.

**Prod:**

<img width="471" alt="Screenshot 2024-01-08 at 11 02 33 am" src="https://github.com/shapeshift/web/assets/97164662/3ae27740-073d-4092-af86-ae85c4c5b6f3">

This branch:

<img width="438" alt="Screenshot 2024-01-08 at 11 03 03 am" src="https://github.com/shapeshift/web/assets/97164662/8914997a-fe18-4296-9cbd-e4e9e01b0c92">

Reply:
The removal was indeed intentional as this error supposedly doesn't exist anymore, see original comment https://github.com/shapeshift/web/pull/5602/files#r1399829076

<img width="721" alt="image" src="https://github.com/shapeshift/web/assets/17035424/d16a2c27-9f71-4cca-9e92-38735f966410">

However it seems like the docs are wrong and the endpoint still returns it, nice catch @0xApotheosis 🙏🏽 

<img width="568" alt="image" src="https://github.com/shapeshift/web/assets/17035424/6709b9b6-9293-4b24-9891-6b71ffabc6ad">

See [docs spec](https://github.com/cowprotocol/services/blob/49c21ed60537d78bcf143e2e2005eea6dc860e34/crates/orderbook/openapi.yml#L1079) vs. [the actual error path at runtime](https://github.com/cowprotocol/services/blob/49c21ed60537d78bcf143e2e2005eea6dc860e34/crates/orderbook/src/api/post_quote.rs#L57)

Reply:
f7fef99dc0